bertopic/_bertopic.py

```diff
@@ -16,7 +16,7 @@
 from tqdm import tqdm
 from scipy.sparse import csr_matrix
 from scipy.cluster import hierarchy as sch
-from typing import List, Tuple, Union, Mapping, Any, Callable, Iterable
+from typing import List, Optional, Tuple, Union, Mapping, Any, Callable, Iterable
```

Comment:
You can remove the `Optional` here and all other places you used it. It is currently not used throughout BERTopic and using it now would cause some confusion.